### PR TITLE
Check password while User writes it

### PR DESCRIPTION
**Pull request by akbarjorayev**

### CODE REVIEW

**Review by akbarjorayev — COMMENTED** [review]
Fix the Style for **Phones**!

**Review by akbarjorayev — COMMENTED** [review]
AUTOFOCUS should be in another log in pages as well!

**Review by akbarjorayev — COMMENTED** [review]
Nod Bad but Nod Good!

File: src/auth/PasswordForm.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.conditions {
```
[the remark this answers is not in the document]
Comment (akbarjorayev) [reply src/auth/PasswordForm.css:1]: Ok

File: src/auth/PasswordForm.js [hunk 2]
```diff
@@ -19,41 +20,46 @@ const PasswordForm = ({ temporaryToken }) => {
...
-      setPasswordError(
-        "Parol 8 ta belgidan kam bo'lmasligi kerak va kamida 1 ta kichik harf, 1 ta bosh harf va 1 ta raqam qatnashishi zarur"
-      );
-      return false;
-    } else {
-      setPasswordError('');
-      return true;
-    }
-  };
+    const containsNumber = /\d/.test(password);
```
[the remark this answers is not in the document]
Comment (akbarjorayev) [reply src/auth/PasswordForm.js:37]: Ok 👍

**Review by akbarjorayev — COMMENTED** [review]
Good. CSS folder should be created

**Review by nazarovctrl — APPROVED** [review]
LGTM

**Pull request merged**